rosdep/python.yaml

```diff
@@ -9521,6 +9521,23 @@ python3-squaternion-pip:
   ubuntu:
     pip:
       packages: [squaternion]
+python3-sshtunnel:
+  debian: [python3-sshtunnel]
+  fedora: [python3-sshtunnel]
+  ubuntu:
+    '*': [python3-sshtunnel]
```

Comment:
@mjeronimo I'm not much a python user, but this doesn't look quite right to me. I'm not sure why we have something called `python3-XYZ` and then `XYZ` under pip. Looking over this file, I don't see any analogs to this setup which makes me think its incorrectly formatted.

Reply:
Frankly, I'm not sure whether this is correct, but I've seen this in the python.yaml. Here is an example with the rosdep key `python-future`: https://github.com/ros/rosdistro/blob/1962b88f2000d984a9bfa48a04670c076482c630/rosdep/python.yaml#L1851

Reply:
@SteveMacenski I've decided to just remove the pull from pip. I hoped to get both the system's package distributor (e.g. apt on ubuntu) and pip inside the python.yaml. However, I see no way to do that without using the wildcard operator, which is too powerful (no package for bionic: https://github.com/ros/rosdistro/actions/runs/4141872809). So I just removed pip: https://github.com/ros/rosdistro/pull/36071/commits/f6e326ac42ae761db1877e8687289bfabd843c09